src/main/java/spireMapOverhaul/zones/thieveshideout/patches/HighlightForcedEventRoomPatch.java

```diff
@@ -0,0 +1,58 @@
+package spireMapOverhaul.zones.thieveshideout.patches;
+
+import basemod.ReflectionHacks;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.map.MapRoomNode;
+import com.megacrit.cardcrawl.vfx.FlameAnimationEffect;
+import spireMapOverhaul.zones.thieveshideout.rooms.ForcedEventRoom;
+
+import java.util.ArrayList;
+
+public class HighlightForcedEventRoomPatch {
```

Comment:
This is basically just a patch copy of the burning elite vfx. I'd written something like this before so it was easy to adapt.

Happy to swap this out for something else if we decide on a standard like the `!` idea you mentioned.

Reply:
Personally I think an exlamation mark has less room for misunderstandings since the burning effect is already associated with keys. However I don't care overmuch, if it's too much work we can just go with this